Seneca-CDOT/telescope
# Initial work for PWA

## Issue This PR Addresses
Addresses #801 



## Type of Change




## Description


How to test:
- pull this branch
- `npm install`
- `npm run clean`
- `npm run build`
- `npm run serve`
- go to `http://localhost:9000`

You should see:
![Annotation 2020-03-05 151434](https://user-images.githubusercontent.com/36822198/76022181-a286dd00-5ef4-11ea-99a0-ebd7f1c2a7e8.png)

The (+) button lets you Install Telescope as a desktop app but it won't actually load any blogs
![Annotation 2020-03-05 151434_](https://user-images.githubusercontent.com/36822198/76022406-19bc7100-5ef5-11ea-8737-85969ffd74d8.png)

## Checklist
